centralledger/chart-handler-admin-transfer/configs/default.json

```diff
@@ -465,5 +465,7 @@
                 }
             }
         }
-    }
+    },
+    "ADDITIONAL_PARTICIPANT_LEDGER_ACCOUNT_TYPES": [],
+    "SETTLEMENT_MODELS": [ "DEFERREDNET" ]
```

Comment:
Shouldn't both lines 469 and 470 be removed?

Don't forget to remove the end `,` once making changes.